src/components/MAPI/apps/AppList/utils.tsx

```diff
@@ -164,52 +123,63 @@ function compareAppCatalogs(
 export function mapAppCatalogsToFacets(
   appCatalogs: applicationv1alpha1.IAppCatalog[] = [],
   selectedAppCatalogs: Record<string, boolean> = {},
-  appCatalogErrors: Record<string, string> = {}
+  error: string = ''
 ): IFacetOption[] {
-  return appCatalogs.sort(compareAppCatalogs).map((appCatalog) => {
+  const sortedAppCatalogs = appCatalogs.sort(compareAppCatalogs);
+
+  const facetOptions: IFacetOption[] = [];
+  for (const appCatalog of sortedAppCatalogs) {
+    // TODO(axbarsan): Remove this once https://github.com/giantswarm/giantswarm/issues/17490 is done.
+    if (appCatalog.metadata.name === 'helm-stable') continue;
```

Comment:
We filter out the `Helm Stable` catalog on the client side until the server-side part is done